components/brave_component_updater/browser/brave_component.cc

```diff
@@ -36,10 +36,9 @@ void BraveComponent::Register(const std::string& component_name,
                           weak_factory_.GetWeakPtr(),
                           component_id);
 
-  delegate_->Register(component_name_,
+  delegate_->Register(component_id_, component_name_,
```

Comment:
Remove component_id changes, replicate deriving the id from the public key like desktop does

Reply:
done